Classes/Domain/AbstractAggregateRoot.php

```diff
@@ -27,20 +27,19 @@
     /**
      * @var string
      */
-    private $aggregateIdentifier;
+    protected $aggregateIdentifier;
```

Comment:
This can`t be `private` because of our `unserialize` trick in `EventSourcedRepository:: findByIdentifier()`. But I think this is OK and it allows aggregate implementations to set the id in a custom constructor